sources/lib/reasoners/intervalCalculus.ml

```diff
@@ -1663,9 +1740,10 @@ let add =
     try
       let env = {env with new_uf} in
       if is_num r then
+        let env, eqs = add_used_by r env in
         init_monomes_of_poly are_eq env
-          (poly_of r) SX.empty Explanation.empty
-      else env
+          (poly_of r) SX.empty Explanation.empty, eqs
```

Comment:
You can start by doing ` let env = init_monomes ... in add_used_by ...` directly